src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Implementation/Policy/NewAzurePolicyDefinition.cs

```diff
@@ -75,7 +75,7 @@ public class NewAzurePolicyDefinitionCmdlet : PolicyCmdletBase
         /// </summary>
         [Parameter(Mandatory = false, ValueFromPipelineByPropertyName = true, HelpMessage = PolicyHelpStrings.NewPolicyDefinitionModeHelp)]
         [ValidateNotNullOrEmpty]
-        public PolicyDefinitionMode? Mode { get; set; }
+        public PolicyDefinitionMode Mode { get; set; }
```

Comment:
Why is this not simply a string?  We suggest using a string and a TabCompleter for these kinds of arguments, so that users are not stuck if a new value is added srevice side until the tools are updated.

Reply:
@markcowl resolved